promises.cf.in

```diff
@@ -55,15 +55,15 @@ body common control
       # modules in which you MUST provide package modules used to generate
       # software inventory reports. You can also provide global default package module
       # instead of specifying it in all package promises.
-    debian.!disable_inventory_package_refresh::
+    (debian|windows).!disable_inventory_package_refresh::
```

Comment:
mmmmm, I think that you need to have this default package module set for windows only if its the appropriate version, else you won't have a module body defined.

Reply:
windows.(cfengine_3_12|cfengine_3_13|cfengine_3_14|cfengine_3_15).!cfengine_3_12_0::

Reply:
And actually, thinking again, shouldnt it be `!(cfengine_3_12_0|cfengine_3_12_1)` since 3.12.1 is out now?

Reply:
Good point, but I'd like to combine it with minimum_version, like this:
```
# for CFEngine 3.12.{2,3,4,...}
    windows.cfengine_3_12.!(cfengine_3_12_0|cfengine_3_12_1)::
          package_inventory =>...;
          package_module =>...;
# for CFEngine 3.{13,14,15,...}.x
@if minimum_version(3.13)
    windows::
          package_inventory => ...;
          package_module => ...;
@endif
```
This causes copy-paste programming, but assuming that `cfengine_3_12` class is never defined together with `minimum_version(3.13)`  macro, there should be no errors in policy.